src/app/learnocaml_index_main.ml

```diff
@@ -257,8 +367,9 @@ let exercises_tab token : tab_handler =
               let exos = format_exercise_list grp.Exercise.Index.contents in
               Manip.Ev.onclick title
                 (fun _ ->
-                   ignore (Manip.toggleClass title "collapsed");
-                   false);
+                  update_fragment "expand" id;
+                  ignore (Manip.toggleClass title "collapsed");
+                  false);
```

Comment:
This two lines were untouched by your PR except the indentation (spaces),
it might be better to restore the previous indentation so that the diff is not ambiguous (i.e., suggesting that the lines changed, while they didn't)